lib/origen_testers/api.rb

```diff
@@ -148,7 +148,18 @@ def c1(msg, options = {})
     def c2(msg, options = {})
       c1(msg, options)
     end
-
+    
+    def c3(msg, options = {}, line_number)
```

Comment:
Per the comments here: https://github.com/Origen-SDK/origen/pull/250#issuecomment-375242047

I think this should be implemented as branch within the c1/cc method, rather than introducing a new method.